features/_trustedboot/file.include/usr/bin/repart-esp-disk

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -eufo pipefail
+
+efi_part_dev="$(realpath "$1")"
+efi_part_name="$(basename "$efi_part_dev")"
+
+disk_name="$(basename "$(realpath "/sys/class/block/$efi_part_name/..")")"
+disk_dev="/dev/$disk_name"
+
+echo | sfdisk "$disk_dev"
```

Comment:
Are we creating an empty partition here? Maybe some expiation could be helpful why.

Reply:
This is to handle a resized disk, so when our small disk image got imaged to a larger disk. now the secondary GPT table won't be at the end of the disk anymore.

we are essentially applying an identity transform on the disk with sfdisk, however, even when not making changes sfdisk will always fix up the secondary GPT table to be at the end of the disk.

Reply:
added a comment to clarify